manifests/kustomize/base/cache-deployer/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kubeflow
```

Comment:
remove the line

Reply:
currently the cache kustomization.yaml and deployer kustomization.yaml are directly added on presubmit tests(https://github.com/kubeflow/pipelines/pull/3036/files#diff-fd55bf48b17b91abb29e5c427891090eR8). Namespace is needed here. In the future when this component is part of the pipeline this line will be removed.

Reply:
You could just add to that test kustomization file then.